.github/workflows/pr-labeler-kind.yaml

```diff
@@ -16,4 +20,39 @@ jobs:
       - name: Label PR based on branch name
         uses: actions/labeler@v5
         with:
-          configuration-path: .github/config/labeler.yml
\ No newline at end of file
+          configuration-path: .github/config/labeler.yml
+  size-labeler:
+    runs-on: ubuntu-latest
+    name: Label the PR size
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_label: 'size/xs'
+          xs_max_size: '10'
+          s_label: 'size/s'
+          s_max_size: '100'
+          m_label: 'size/m'
+          m_max_size: '500'
+          l_label: 'size/l'
+          l_max_size: '10000'
```

Comment:
I know `10000` LoC is our current maximum, but should we talk again about that threshold as it is quite large?

Reply:
Yes, but please lets not do this in this PR. Would you mind creating a separate issue for this?

Reply:
https://github.com/open-component-model/ocm/issues/1107